docs/get-started.md

```diff
@@ -29,34 +29,36 @@ Usually, GitHub will enable GitHub Pages as soon as a `gh-pages` branch is detec
 
 1. Go to your repository settings page
 1. Go to the "Pages" sub-section on the left
-1. Under "Source", change "None" to `gh-pages`
-1. In the folder dropdown, select `/(root)`
-1. Skip the "Theme Chooser" option 
+1. Under "Source", change "None" to "Deploy from a branch"
+1. In the Branch dropdown, select `gh-pages` and `/(root)`
 1. Click on "Save"
 
-After saving, you will see confirmation text "Your site is ready to be published at...". For more information on enabling GitHub Pages, see the article on the GitHub website: [Configuring a publishing source for your GitHub Pages site](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site).
+After saving, you will see confirmation text "Your site is live at..." at the top of the page. For more information on enabling GitHub Pages, see the article on the GitHub website: [Configuring a publishing source for your GitHub Pages site](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site).
```

Comment:
The GitHub Pages site URL appears at the top of the Pages Settings page like it used to but it has different wording now. Also it is shown when going to edit the repository metadata (like description, URL, tags) from the main repo page. There is now a checkbox named "Use your GitHub Pages website". Should I mention this as well?